### PR TITLE
feat(commands): add a command for a list of "good to know" items for job applications, particularly juniors

### PR DESCRIPTION
**Pull request by mwskwong**
The list of items is open for discussion. Feel free to comment, add, update, or remove some of the items.

I added [`dedent`](https://www.npmjs.com/package/dedent) for better DX. Please let me know if you prefer not to do this in this PR, or use `dedent` to all other commands at once.

### CODE REVIEW

**Review by vcarl — APPROVED** [review]
Code-wise this seems good to me! Saw you ask for feedback in the channel, this is good to merge if there isn't any dissent there

**Review by ghardin137 — APPROVED** [review]

File: package.json [hunk 1]
```diff
@@ -21,6 +21,7 @@
   "license": "MIT",
   "dependencies": {
     "date-fns": "3.6.0",
+    "dedent": "^1.5.3",
```
Comment (kristersd) [inline package.json:24]:
```suggestion
    "dedent": "1.5.3",
```

**Review by kristersd — APPROVED** [review]

**Comment by mwskwong** [comment]
@vcarl @ghardin137 @kristersd There seem to be some issues with the build pipeline. Could you have a look?

**Comment by vcarl** [comment]
Sorry @mwskwong hadn't seen this! That's a known issue (#216), I ignored it for a while cuz it was ~ just me contributing hehe. Has been more and more confusing so I killed the thing causing it 👍 tyty

**Comment by mwskwong** [comment]
@vcarl I see. I guess your side need to re-trigger the checks or force merge this PR then, since I don't have permission to do so.

**Pull request merged**